tests/common/macro_support.hpp

```diff
@@ -173,7 +173,7 @@ inline std::vector<Name> sort_names( std::vector<Name>&& names ) {
    { \
       eos::chain::SignedTransaction trx; \
       trx.scope = sort_names( { "eos", #account } ); \
-      trx.emplaceMessage(config::EosContractName, vector<types::AccountPermission>{{#account, "active"}}, \
+      trx.emplaceMessage(config::EosContractName, vector<types::AccountPermission>{}, \
```

Comment:
Also lines 150, 164... Why remove the declared authorizations? Doesn't this make tests break? If not, something is wrong.